server/plugin/api.go

```diff
@@ -233,6 +234,37 @@ func (p *Plugin) handleUnlinkProject(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// handleGetUserAccountDetails provides user details
+func (p *Plugin) handleGetUserAccountDetails(w http.ResponseWriter, r *http.Request) {
+	mattermostUserID := r.Header.Get(constants.HeaderMattermostUserIDAPI)
+
+	userDetails, err := p.Store.LoadUser(mattermostUserID)
+	if err != nil {
+		p.API.LogError(constants.ErrorDecodingBody, "Error", err.Error())
```

Comment:
if we are loading the user why there is error for decoding body rather than failed to decode error